pillar.example

```diff
@@ -51,13 +52,26 @@ packages:
       - diff-lcs
       - kitchen-vagrant
       - kwalify
+
   snaps:
     wanted:
       - hello-world
     classic:
       - test-snapd-hello-classic
     unwanted:
       - goodbye-world
+
+  npms:
+    required:
+    # pkgs:
+    #   - some_package_you_need
+      sls: ['nodejs.ppa']
+    wanted:
+      - hello-world-npm
+      - sax
```

Comment:
It would be great to have an example here of how to supply the package version as well.  Again, using the example from the upstream docs, something like `coffee-script@1.0.1`.